### PR TITLE
V2.6.7 beta

### PR DESCRIPTION
**Pull request by dvdplm**

### CODE REVIEW

File: secret-store/src/key_storage.rs [hunk 1]
```diff
@@ -297,7 +273,7 @@ pub mod tests {
 	use parking_lot::RwLock;
 	use self::tempdir::TempDir;
 	use ethkey::{Random, Generator, Public};
-	use kvdb_rocksdb::Database;
+use kvdb_rocksdb::{Database, DatabaseConfig};
```
Comment (ordian) [inline secret-store/src/key_storage.rs]:
```suggestion
	use kvdb_rocksdb::{Database, DatabaseConfig};
```

**Comment by ordian** [comment]
```
error[E0433]: failed to resolve: use of undeclared type or module `parity_crypto`
   --> ethcore/types/src/transaction/transaction.rs:650:8
    |
650 |             Err(parity_crypto::publickey::Error::InvalidSignature) => {}
    |                 ^^^^^^^^^^^^^ use of undeclared type or module `parity_crypto`
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+Parity Ethereum v2.6.7-beta is a patch release that adds Istanbul hardfork
+block numbers for POA and xDai networks, implements ECIP-1056 and implements
+EIP-2384/2387 - Muir Glacier.
+
+The full list of included changes:
+* ethcore/res: activate agharta on classic 9573000 (#11331)
+* Istanbul HF in xDai (2019-12-12) (#11299)
+* Istanbul HF in POA Core (2019-12-19) (#11298)
+* Istanbul HF in POA Sokol (2019-12-05) (#11282)
+* Enable EIP-2384 for ice age hard fork (#11281)
```
Comment (niklasad1) [inline CHANGELOG.md]: I think we should add #11332 and #11335 to the changelog

File: ethcore/src/miner/miner.rs [hunk 3]
```diff
@@ -1709,6 +1709,30 @@ mod tests {
 		assert_eq!(miner.prepare_pending_block(&client), BlockPreparationStatus::NotPrepared);
 	}
 
+	#[test]
+	fn should_reject_local_transaction_with_invalid_chain_id() {
+		let spec = spec::new_test();
```
Comment (niklasad1) [inline ethcore/src/miner/miner.rs]: this fails should probably be `spec::Spec::spec::next_test()`

**Comment by soc1c** [comment]
can we also get #11338 in?

**Comment by dvdplm** [comment]
> can we also get #11338 in?

Sure, I don't see why not. For both stable and beta?

**Comment by soc1c** [comment]
yes, thanks.

File: CHANGELOG.md (hunk 2 above)
Comment (dvdplm) [reply CHANGELOG.md]: Definitely.

**Pull request merged**